JustSaying.IntegrationTests/WhenRegisteringASqsSubscriber/WhenRegisteringLongNameMessageTypeTopicSubscriber.cs

```diff
@@ -0,0 +1,23 @@
+﻿using JustSaying.Messaging.MessageHandling;
```

Comment:
This test is missing from csproj. @SamHWhale.
PR seems fine. Can you please update your PR and I'll merge it.